python/ray/train/_internal/data_config.py

```diff
@@ -26,21 +25,30 @@ class DataConfig:
 
     def __init__(
         self,
-        datasets_to_split: Optional[List[str]] = None,
+        datasets_to_split: Union[Literal["all"], List[str]] = "all",
         execution_options: Optional[ExecutionOptions] = None,
     ):
         """Construct a DataConfig.
 
         Args:
-            datasets_to_split: The list of dataset names to split between workers.
-                By default, only the "train" dataset will be split.
+            datasets_to_split: Specifies which datasets should be sharded among workers.
```

Comment:
`sharded` -> `split`? It'd better to be consistent with the terminology in docstring.